pubspec.yaml

```diff
@@ -5,7 +5,7 @@ version: 5.0.2
 homepage: https://github.com/marcglasberg/back_button_interceptor
 
 environment:
-  sdk: '>=2.13.0 <3.0.0'
+  sdk: '>=2.13.0 <=3.0.0'
```

Comment:
Wouldn't you want to leave this at below `3.0.0` so that it's guaranteed to be compatible with versions of Dart below that major version?